src/translations/views/TranslationsPages.tsx

```diff
@@ -60,57 +64,88 @@ const TranslationsPages: React.FC<TranslationsPagesProps> = ({
       navigate("?", true);
     }
   };
+
   const onDiscard = () => {
     navigate("?", true);
   };
 
   return (
-    <TypedUpdatePageTranslations onCompleted={onUpdate}>
-      {(updateTranslations, updateTranslationsOpts) => {
-        const handleSubmit = (
-          fieldName: PageTranslationInputFieldName,
-          data: string
-        ) => {
-          updateTranslations({
-            variables: {
-              id,
-              input: getParsedTranslationInputData({ data, fieldName }),
-              language: languageCode
-            }
-          });
-        };
-        const translation = pageTranslations?.data?.translation;
+    <TypedUpdatePageTranslations
```

Comment:
Tbh I think that if we're already pushing more stuff here in jsx and we are certain we want to get rid of those Typed queries / mutations, then this could be the right time to move all this outside of return with a regular useMutation. @jwm0 wdyt?

Reply:
It should but this is out of scope for this task, this should be a maintenance task